File: HoWo7/PhoneCompany/Vivo.py
from Phone import Smartphone

class Vivo(Smartphone):
    def __init__(self,model:str, sizeDisplay, akb: int, typeOfAntenna:str, osVerion:str, price:int):
        super().__init__(model, sizeDisplay, akb, price, ("Android " + osVerion) , typeOfAntenna)

    def __str__(self) ->str:
        return "Vivo " + super().__str__()

File: HoWo5_6/PhoneCompany/Vivo.java
package HoWo5_6.PhoneCompany;

import HoWo5_6.Phone.Smartphone;

public class Vivo extends Smartphone{

    public Vivo(String model, double sizeDisplay, int akb, String typeOfAntenna, String osVerion, int price) {
        super(model, sizeDisplay, akb, price, ("Android " + osVerion) , typeOfAntenna);
    }

    @Override
    public String toString() {
        return "Vivo "+super.toString();
    }

}
